lib/pengine/pe_output.c

```diff
@@ -513,6 +513,36 @@ pe__cluster_dc_xml(pcmk__output_t *out, va_list args) {
     return 0;
 }
 
+int
+pe__cluster_maint_mode_html(pcmk__output_t *out, va_list args) {
+    gboolean maint_mode = va_arg(args, gboolean);
+
+    if (maint_mode) {
+        xmlNodePtr node = pcmk__output_create_xml_node(out, "li");
+
+        pcmk_create_html_node(node, "span", NULL, "bold", "DISABLED");
+        pcmk_create_html_node(node, "span", NULL, NULL,
+                              " (the cluster will not attempt to start, stop, or recover services)");
+    } else {
+        out->list_item(out, NULL, "Resource management enabled");
+    }
```

Comment:
I'm thinking maintenance mode should be in the list with the other options in html (as "Resource management: ...") similar to how XML does. Only text/curses should display it as a banner separate from the options list.

That may complicate the message model. One possibility would be to remove maintenance mode as a "section", and display it with options for xml/html and display it unconditionally for text/curses. Another possibility would be to use a single custom message (for options), and have it take booleans for whether to display maintenance mode and whether to display the other options. Whatever works is fine.

Reply:
Is there a reason to not just unconditionally show it all the time, for all output formats?

Reply:
That would be fine.